music_server.py

```diff
@@ -1,14 +1,32 @@
-from flask import Flask
-from waitress import serve
 
+from flask import Flask, send_from_directory, abort
+import os
+from waitress import serve
 import logging
 
 app = Flask(__name__)
-
+# Set the path to the music folder in the working directory
+app.config['MEDIA_FOLDER'] = os.path.join(os.getcwd(), 'Music')
 @app.route('/')
 def server_home_page():
 	return "This is the server's home page. Nothing interesting to see here."
+@app.route('/play/<album>/<track>')
+#Use the following url to play the tracks within the albums.e %20 for space
+# :http://127.0.0.1:5000/play/Album%203/Track%204.mp3
+#For example: The above url will play track 4 from album 3.
+def play_song(album, track):
+    # Sanitize input to match directory and file naming conventions
+    valid_albums = ['Album 1', 'Album 2', 'Album 3']
```

Comment:
Instead of using a hard-coded list of the album names, we should be fetching a list of the names from the "Music" folder. Can you please change this?

Reply:
I like it! This is a better idea. I made the suggested changes and updated the code. Thanks!

Reply:
Great! Thank you!